### PR TITLE
Large Transaction Client

### PR DESCRIPTION
**Pull request by rado0x54**
Implemented `signLarge` Transaction execution on `simpleCryptid` and `controlledCryptid`.

### CODE REVIEW

File: client/src/api/simpleCryptid.ts [hunk 1]
```diff
@@ -19,14 +21,34 @@ export class SimpleCryptid extends AbstractCryptid {
...
+        this.did,
+        this.signer.publicKey,
+        [[this.signer, []]]
+      );
+
+      checkTxSize(wrappedTransaction);
+
+      return wrappedTransaction;
+    } catch (e) {
+      if (e instanceof RangeError) {
```
Comment (william-brooks) [inline client/src/api/simpleCryptid.ts]: I think the type check here isn't working (at least not in the wallet)... always ends up in the else below (the e.message is correctly the range error)

File: client/src/lib/solana/transactions/largeExecute.ts [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  if (nrOfOverflowInstructions > 0) {
+    const expandInstruction = await createExpand(
+      [],
+      mappedOverheadInstructions.map(instruction => new InstructionOperation({
+        push: instruction
+      })),
+      didPDAKey,
+      TRANSACTION_SEED,
+      true,
+      signersNormalized[0], // TODO: Brett why is this not array? (as with propose). e.g. signers
```
Comment (william-brooks) [inline client/src/lib/solana/transactions/largeExecute.ts:106]: @Buzzec feedback here?

File: client/src/lib/solana/transactions/largeExecute.ts [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    );
+
+    setupTransactions.push(expandTransaction)
+  }
+
+  // Build execute Transaction.
+  const execute = await createExecute(
+    didPDAKey,
+    TRANSACTION_SEED,
+    signersNormalized[0], // TODO: Check in with Brett
```
Comment (william-brooks) [inline client/src/lib/solana/transactions/largeExecute.ts:124]: @Buzzec ?

File: client/src/api/abstractCryptid.ts [hunk 4]
```diff
@@ -182,4 +187,18 @@ export abstract class AbstractCryptid implements Cryptid {
...
     return [];
   }
+
+  async listPendingTx(): Promise<PublicKey[]> {
+    const address = await deriveDefaultCryptidAccount(this.did);
+
+    return this.options.connection
+      .getProgramAccounts(CRYPTID_PROGRAM_ID, {
+        filters: [
+          // TODO: Confirm these filters are correct
```
Comment (william-brooks) [inline client/src/api/abstractCryptid.ts:197]: @Buzzec can you check this too please to see if the filters are correct

File: client/src/api/simpleCryptid.ts (hunk 1 above)
Comment (rado0x54) [reply client/src/api/simpleCryptid.ts]: with your new logic this code should not be relevant any longer.

**Pull request merged**